targets/simple_switch_grpc/switch_runner.cpp

```diff
@@ -294,15 +295,16 @@ SimpleSwitchGrpcRunner::init_and_start(const bm::OptionsParser &parser) {
 
   // check if CPU port number is also used by --interface
   // TODO(antonin): ports added dynamically?
-  if (cpu_port >= 0) {
+  if (cpu_port > 0) {
```

Comment:
it seems that 0 is still a valid value for dataplane ports (at least I hope it is the case because many bmv2 users rely on this), but that it is no longer a valid value for the cpu_port for simple_switch_grpc
in this case I believe you should update main.cpp to 1) indicate in the help string for --cpu-port that 0 is not valid, 2) return an error if the user has passed a value of 0, and 3) probably add a static_cast when `SimpleSwitchGrpcRunner::get_instance` to make the intent clear.

Reply:
alternatively you can use `boost::optional<uint32_t>` in the `SimpleSwitchGrpcRunner` interface if you think `boost::optional` is an acceptable dependency and keep 0 as a valid value

Reply:
0 is still valid for data plane ports. In fact it is the default value for incoming/outgoing port in dataplane_interface.proto. This means that we cannot use 0 as a cpu_port. I updated main.cpp accordingly.